src/util/dsp/LogSlider.cpp

```diff
@@ -0,0 +1,58 @@
+#include "LogSlider.hpp"
+#include <math.h>
+namespace otto::util::dsp {
+  LogSlider::LogSlider()
+  {
+  }
+  LogSlider::~LogSlider()
+  {
+  }
+
+  /*
+    BELOW IS THE CORE DEAL OF THE LOGARITHMIC SLIDER
```

Comment:
I plan to set up [standardese](https://github.com/foonathan/standardese) again for documentation, so we should strive to use that syntax everywhere.
The basics is:
```
/// Documentation comments like this
/**
 * Or like this
 */
```
Documentation will be parsed as markdown.

In this case, just wrap it in doc comments, and put the code/table in \`\`\` code blocks.
But nice documentation!

Also, documentation should go in header files. I know this is partly an explanation of the implementation, so maybe split it up